scripts/validate-front-matter/script.py

```diff
@@ -8,6 +8,7 @@
 try:
     from yaml import CLoader as Loader
 except ImportError:
+    print("Warning: Using slower Python YAML loader. Install LibYAML for faster performance.")
```

Comment:
To be able to check whether we have the LibYAML dependency installed. Currently we do, but on a future runner we may not.